app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (OutdatedVaccineError,
+                        NotVaccinatedError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+
+        expiration_date = visitor["vaccine"]["expiration_date"]
+        if expiration_date and expiration_date < datetime.date.today():
+            raise OutdatedVaccineError("Visitor's vaccine "

```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line